ddtrace/internal/telemetry/data/telemetry_request.py

```diff
@@ -0,0 +1,75 @@
+import time
+from typing import Any
+from typing import Dict
+from typing import TypedDict
+
+from ...runtime import get_runtime_id
+from .application import APPLICATION
+from .application import Application
+from .host import HOST
+from .host import Host
+from .payload import Payload
+
+
+RequestBody = TypedDict(
+    "RequestBody",
+    {
+        "tracer_time": int,
+        "runtime_id": str,
+        "api_version": str,
+        # seq_id (sequence id) should be incremented every time a telemetry
+        # request is sent to the agent. This field will be used to monitor
+        # dropped payloads and reorder requests on the backend
+        "seq_id": int,
+        "application": Application,
+        "host": Host,
+        "payload": Dict[Any, Any],
+        "request_type": str,
+    },
+)
+"""
+Contains all the body fields required by v1
+of the Telemetry Intake API
+"""
+
+TelemetryRequest = TypedDict(
+    "TelemetryRequest",
+    {
+        "headers": Dict[str, str],
+        "body": RequestBody,
+    },
+)
+"""
+Contains all the header and body fields required
+to send a request to v1 of Telemetry Intake Service
+"""
```

Comment:
these multiline strings seem disconnected, what are they meant to relate to?

Reply:
oh, this is meant for `TelemetryRequest`.

Instead you can just do a normal comment on the line before.

```python
# Contains all the header and body fields required
# to send a request to v1 of Telemetry Intake Service
TelemetryRequest = TypedDict()
```

Reply:
Yes, this is how I set docstrings for all the typed dicts in this pull request. I'll make this change for Host, Application, Integration, Payload and Dependency dicts as well